Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -83,17 +83,20 @@ DECLARE_SOA_COLUMN(Z, z, float);
 DECLARE_SOA_COLUMN(Snp, snp, float);
 DECLARE_SOA_COLUMN(Tgl, tgl, float);
 DECLARE_SOA_COLUMN(Signed1Pt, signed1Pt, float);
-DECLARE_SOA_DYNAMIC_COLUMN(Phi, phi, [](float snp, float alpha) -> float {
-  float phi = std::asin(snp) + alpha;
-  constexpr float twopi = 2.0 * M_PI;
+// FIXME: find a way to clean up node templates so '1.f *' workaround is not needed
+DECLARE_SOA_EXPRESSION_COLUMN(Phi, phi, float, nasin(aod::track::snp) + aod::track::alpha);
```

Comment:
What you have as 'nphi' must be the default. So already renormalized within 0 and 2 pi. 
You can just rename:
phi --> phiraw
nphi --> phi

Reply:
Done.